shell/common/rasterizer.cc

```diff
@@ -270,13 +271,23 @@ sk_sp<SkImage> Rasterizer::DoMakeRasterSnapshot(
                 return;
               }
 
+              GrDirectContext* context = surface_->GetContext();
+
+              // Upper-bound to the max render target size supported by the GPU.
+              // Prefer to proceed with a smaller snapshot over failing with a
+              // null result.
+              auto maxSize = context->maxRenderTargetSize();
```

Comment:
Nit: Underscore case for the `max_size` variable in C++ translation units please.

Reply:
Done 👍